save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/NameValidation.kt

```diff
@@ -0,0 +1,31 @@
+/**
+ * Module that implements for name checks.
+ */
+
+package com.saveourtool.save.validation
+
+private fun String.hasOnlyLettersOrDigitsOrHyphens() = all { it.isLetterOrDigit() || it == '-' }
+
+private fun String.containsForbiddenWords(additionalForbiddenWords: List<String> = emptyList()) = FrontendRoutes.values()
+    .map {
+        it.path.split("/")
+    }
```

Comment:
Maybe extract this to constants too?

Reply:
Makes sense

Reply:
Oh, I actually meant the whole thing. We can have a `fun` or a `val` with getter that would return a list of forbidden tokens and it would contain the logic of splitting and flattening.

Reply:
Ok, done